data/distro.yml

```diff
@@ -794,3 +794,29 @@
         <p>Note: graphical installation of Flatpak apps may not be possible with GNU Guix.</p>
       </li>
     </ol>
+    
+- name: "PCLinuxOS"
+  logo: "pclinuxos.svg"
+  info: >
+    <ol class="distrotut">
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>The flatpak package is available in the PCLinuxOS repositories, it can be installed using Synaptic Package Manager or by running the following in a terminal:</p>
+        <pre><code>
+          <span class="unselectable">$</span> su
+          <span class="unselectable">#</span> apt-get install flatpak
+        </code></pre>
+      </li>
+      <li>
+        <h2>Add the Flathub repository</h2>
+        <p>Flathub is the best place to get Flatpak apps. To enable it, run:</p>
+        <pre><code>
+          <span class="unselectable">$</span> flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
```

Comment:
Also, as I mentioned, if PCLinuxOS does not have sudo, this needs to be tested. It may not work properly if Polkit permissions are also limited.

Reply:
If it does not work properly, we can modify the command to use user's installation (`flatpak --user remote-add`) instead of a system one.

Reply:
I tried the flatpak installation as instructed by them here: 
http://pclinuxoshelp.com/index.php/Installing_Software#Using_Flatpak  
After the reboot I had no problems.

Reply:
Ok, thanks!